pkg/did/client.go

```diff
@@ -344,8 +349,16 @@ func (c *Client) buildUpdateRequest(did string, sidetreeConfig *models.SidetreeC
 		return nil, err
 	}
 
+	revealValue := updateDIDOpts.RevealValue
+
+	// TODO: client should be managing reveal value, this defaulting here is just temporary convenience (issue-246)
```

Comment:
This todo says that issue #246 is for solving this problem, but your PR says that it closes issue #246. Is issue #246 closed in this PR, or is it future work?

Reply:
good catch, updated numbers in the pull request #256 is future work that we discussed during 1.6 client work planning